clijs/test/commands/receipt.test.ts

```diff
@@ -0,0 +1,25 @@
+import type { Hex, ProcessedReceipt } from "@nilfoundation/niljs";
+import { describe, expect } from "vitest";
+import { CliTest } from "../setup.js";
+
+// To run this test you need to run the nild:
+// nild run --http-port 8529
+describe("receipt:get_receipt", () => {
+  CliTest("tests getting receipts", async ({ runCommand, smartAccount }) => {
+    const txHash = (
+      await runCommand([
+        "smart-account",
+        "send-tokens",
+        smartAccount.address,
+        "--amount",
+        "100",
+        "--feeCredit",
+        10_000_000_000_000 as unknown as string,
+      ])
+    ).result as Hex;
+    expect(txHash).toBeTruthy();
+
+    const r = (await runCommand(["receipt", txHash])).result as ProcessedReceipt;
+    expect(r.success).toBeTruthy();
```

Comment:
Let's also check that `output` contains the string `Receipt data:`.
By the way, I guess in the case of `--quiet` we only need to print the json itself.

Reply:
fixed. also fixed for the `block` command

Reply:
Doesn't [this](https://github.com/oclif/test/blob/0e9113800a3edb99779316a68dc91d6f004d4aa1/test/run-command.test.ts#L13) approach work?

Reply:
I did some research, and found out that without `disableConsoleIntercept` in `vitest`, intercepting the output in `runCommand` doesn't work. They are in conflict. We need to figure out how to make them work together correctly.

Reply:
Hmm. The [documentation](https://www.npmjs.com/package/@oclif/test) for `oclif/test` explicitly says to use `disableConsoleIntercept` for `vitest`.

Reply:
#866